### PR TITLE
feat: 🍰 Show Supported Image Formats Implemented

### LINKED ISSUE #1949: 🚀 [Feature] Show supported image formats in editor
> [datenbrei](https://github.com/datenbrei) **Authored by [datenbrei](https://github.com/datenbrei)**
_<time datetime="2019-10-18T06:28:51Z" title="Friday, October 18th 2019, 8:28:51 am +02:00">Oct 18, 2019</time>_

---

## :rocket: Feature
It would be selfexplaining, if the supported image formats would show of in the editor, as long as there is no image added. This way it would be selfexplaining what formats are supported

### Design & Layout
bildformate

### PR DESCRIPTION
**Pull request by dasari810**
## 🍰 Pullrequest

 to show the supported image formats in the ImageUploader of the editor 

Screenshot from 2020-10-26 00-11-49

### Issues

- fixes #1949 

### Todo

- [X]  the text should be converted to other languages

### CODE REVIEW

**Comment by dasari810** [comment]
@Tirokk  can you help me finding out ... why the checks are failing ?

File: webapp/locales/pt.json [hunk 1]
```diff
@@ -252,7 +252,8 @@
     "newPost": "Criar uma nova publicação",
     "success": "Salvo!",
     "teaserImage": {
-      "cropperConfirm": "Confirmar"
+      "cropperConfirm": "Confirmar",
+      "supportedFormats": "Insira uma imagem do formato de arquivo JPG, PNG ou GIF"
```
Comment (ftonato) [inline webapp/locales/pt.json:256]:
```json
"supportedFormats": "Insira uma imagem do formato JPG, PNG ou GIF"
```

Comment (ftonato) [reply webapp/locales/pt.json:256]: Please, accept my suggestion (make more sense say as I reviewed in Portuguese)

**Review by ftonato — CHANGES_REQUESTED** [review]

File: webapp/locales/pt.json (hunk 1 above)
Comment (ftonato) [reply webapp/locales/pt.json:256]: Please, accept my suggestion (make more sense say as I reviewed in Portuguese)

Comment (dasari810) [reply webapp/locales/pt.json:256]: hey @ftonato  sure , i think a person from Portugal know it well that google translate :wink:

Comment (ftonato) [reply webapp/locales/pt.json:256]: Yeah, maybe (sometimes) ahaha
I'm actually from Brazil, but I live in Portugal, so this is a combination conhecimento de Portuguese o/

**Comment by Mogge** [comment]
Good job. You do not have to do all the translations. We have English as fallback and until now we want to have German as well. But thanks a lot for your effort.
I started the test again. Hopefully it will pass now.

File: webapp/locales/de.json [hunk 2]
```diff
@@ -258,7 +258,8 @@
     "newPost": "Erstelle einen neuen Beitrag",
     "success": "Gespeichert!",
     "teaserImage": {
-      "cropperConfirm": "Bestätigen"
+      "cropperConfirm": "Bestätigen",
+      "supportedFormats": "Fügen Sie ein Bild im Dateiformat JPG, PNG oder GIF ein"
```
Comment (Tirokk) [inline webapp/locales/de.json:262]:
```suggestion
      "supportedFormats": "Füge ein Bild im Dateiformat JPG, PNG oder GIF ein"
```

This is not so formal … 😎

File: webapp/locales/pt.json (hunk 1 above)
Comment (Tirokk) [reply webapp/locales/pt.json:256]: This seems not resolved?
Or is it for you localy?

**Review by Tirokk — CHANGES_REQUESTED** [review]
Hey @dasari810 ,
great! 😁

Your first PR on Ocelot.Social 💫

I have one little change request and be not sure if @ftonato suggestion is solved.

**Comment by dasari810** [comment]
ya @Tirokk  i have done them locally ..  i will push them  :+1:

**Comment by Tirokk** [comment]
I implement the tests these days. They fail, because of wrong settings and secrets … @Mogge @dasari810

**Review by Tirokk — APPROVED** [review]
Wonderful done @dasari810 ! 😎
Great you implemented this feature. 🚀💫

Glad to have you here contributing! 🍀

**Pull request merged**